migtests/tests/analyze-schema/expected_issues.json

```diff
@@ -400,30 +450,79 @@
             "GH": "https://github.com/YugaByte/yugabyte-db/issues/1124"
         },
         {
+            "IssueType": "Unsupported features",
             "ObjectType": "TABLE",
             "ObjectName": "abc",
-            "Reason": "DEFERRABLE unique constraints are not supported yet.",
+            "Reason": "DEFERRABLE constraints not supported yet",
             "SqlStatement": "ALTER TABLE abc\nADD CONSTRAINT cnstr_id\n UNIQUE (id)\nDEFERRABLE;",
-            "Suggestion": "",
-            "GH": "https://github.com/YugaByte/yugabyte-db/issues/1129"
+            "Suggestion": "Remove these constraints from the exported schema and make the neccessary changes to the application to work on target seamlessly",
+            "GH": "https://github.com/yugabyte/yugabyte-db/issues/1709"
         },
         {
+            "IssueType": "Unsupported features",
             "ObjectType": "TABLE",
-            "ObjectName": "Test",
+            "ObjectName": "public.users",
+            "Reason": "DEFERRABLE constraints not supported yet",
+            "SqlStatement": "ALTER TABLE ONLY public.users\n    ADD CONSTRAINT users_email_key UNIQUE (email) DEFERRABLE;",
+            "Suggestion": "Remove these constraints from the exported schema and make the neccessary changes to the application to work on target seamlessly",
+            "GH": "https://github.com/yugabyte/yugabyte-db/issues/1709"
+        },
+        {
+            "IssueType": "Unsupported features",
+            "ObjectType": "TABLE",
+            "ObjectName": "unique_def_test",
+            "Reason": "DEFERRABLE constraints not supported yet",
+            "SqlStatement": "create table unique_def_test(id int UNIQUE DEFERRABLE, c1 int);",
+            "Suggestion": "Remove these constraints from the exported schema and make the neccessary changes to the application to work on target seamlessly",
+            "GH": "https://github.com/yugabyte/yugabyte-db/issues/1709"
+        },
+        {
+            "IssueType": "Unsupported features",
+            "ObjectType": "TABLE",
+            "ObjectName": "unique_def_test1",
+            "Reason": "DEFERRABLE constraints not supported yet",
+            "SqlStatement": "create table unique_def_test1(id int, c1 int, UNIQUE(id)  DEFERRABLE INITIALLY DEFERRED);",
+            "Suggestion": "Remove these constraints from the exported schema and make the neccessary changes to the application to work on target seamlessly",
+            "GH": "https://github.com/yugabyte/yugabyte-db/issues/1709"
+        },
+        {
+            "IssueType": "Unsupported datatypes",
```

Comment:
Let's keep the IssueType like an ENUM? so instead of free text with spaces , we can have 
`unsupported_datatype`, `unsupported_feature` ?